go/libraries/doltcore/doltdb/doltdb.go

```diff
@@ -79,6 +79,12 @@ func DoltDBFromCS(cs chunks.ChunkStore) *DoltDB {
 	return &DoltDB{hooksDatabase{Database: db}, vrw, ns}
 }
 
+// HackDatasDatabaseFromDoltDB unwraps a DoltDB to a datas.Database.
+// Deprecated: only for use in dolt migrate.
+func HackDatasDatabaseFromDoltDB(ddb *DoltDB) datas.Database {
+	return ddb.db
+}
```

Comment:
What's the plan for this?  Feels like these hack calls are just gonna get left in here forever.

Reply:
this is definitely the sketchiest part of this PR... I'll make an effort to clean this up soon.